### ISSUE TITLE
Supporting Enzyme.jl Properly

### ISSUE THREAD

**Issue by avik-pal** [LuxDL/Lux.jl#605]
Currently an empty issue. I will keep updating this with issues that come up while testing with Enzyme.

**Comment by wsmoses** [comment]
@avik-pal the `fast_activation!!` issue is closed, let me know if anything else is blocking there.

**Comment by avik-pal** [comment]
Yes, I have been playing around with it, so I will open any issue upstream as and when they show up.

**Comment by avik-pal** [comment]
After https://github.com/LuxDL/LuxLib.jl/pull/69 and https://github.com/LuxDL/Lux.jl/pull/640 are merged, larger models from Boltz.jl seem to work seamlessly :tada:
